### ISSUE TITLE
How to get all entries of an array via array to latex, in Qiskit, with precision upto 14 decimal places 

### ISSUE THREAD

**Issue by MubasharK** [Qiskit/qiskit-terra#7495]
### What should we add?

I want to ask two questions as I could not find answer to them on this forum and elsewhere. First, what is a precision of an array representing a quantum state which has been evolved via quantum circuit in Qiskit. Second, the command array to latex does not output all entries of the array instead it displays few entries with ellipses in between them.  How can we get complete array with precision up to 14 decimal places (say)?
According to my observations, the precision seems to be 5 decimal places. If it is true, then how one can enhance it? Also, I request you to update the output of array to latex in Qiskit so that all elements of the output tuple are explicitly listed.

**Comment by frankharkins** [comment]
@MubasharK
- To get 14 decimal places of precision, you can pass the argument `precision=14`
- To output more entries, you can set `max_size` to some value that's greater than, or equal to, your array size. If your array is a numpy array, you can set it to that array's `.shape`.
```
from qiskit.visualization import array_to_latex
import numpy as np
arr = np.random.random((50,50))  # max large numpy array
array_to_latex(arr, precision=14, max_size=arr.shape)
```
You can read about this and other features in the [docs](https://qiskit.org/documentation/stubs/qiskit.visualization.array_to_latex.html).

**Comment by MubasharK** [comment]
Thank you Frank Harkins!
Please see the below file. The output of block 4 and 5 is my concern. With including precision=14 in the command array_to_latex one issue has been resolved. But, not all entries of the output array are explicitly listed. The state (array) is initialized using 
from qiskit.quantum_info import Statevector
state = Statevector.from_int(0, 2**7)
state = state.evolve(qc)
#draw using latex
state.draw('latex')
[Multi qubit controlled rotations.pdf](https://github.com/Qiskit/qiskit-terra/files/7829426/Multi.qubit.controlled.rotations.pdf)

**Comment by frankharkins** [comment]
Replace:
```
array_to_latex(state)    -->    array_to_latex(state, max_size=2**7, precision=14)
```
Looks like the ket convention drawer is ignoring max size, I've added a bug report for this (https://github.com/Qiskit/qiskit-terra/issues/7516). Once fixed, I think you should be able to do `state.draw('latex', max_size=2**7)` to get what you want.

**Comment by MubasharK** [comment]
Hello Harkins,

Hope you are well.

The issue  (#7516 <https://github.com/Qiskit/qiskit-terra/issues/7516>) is
still open and I am unable to retrieve all the entries using the
command  state.draw('latex',
max_size=2**7). Is there any progress about this?


Best,
Mubashar

**Comment by kdk** [comment]
Closing as it looks like this was resolved as part of #7516 . Please feel free to re-open if you're still seeing this issue.
